docs/Development-Guides/Providers.md

```diff
@@ -200,3 +205,17 @@ Xero | `xero` | <span style="color:goldenrod">Assisted</span> | <span style="col
 Zoho Payroll | `zoho_payroll` | <span style="color:goldenrod">Assisted</span> | <span style="color:goldenrod">Assisted</span>
 
 **these assisted providers are **Pay Enabled**, meaning initial connection time is 14 days instead of 6 weeks*
+
+<!--
+type: tab
+title: ATS
+-->
+<!-- theme: info -->
+> Assisted Connect is not currently available for ATS integrations.
```

Comment:
Is this even worth mentioning?

Reply:
I'm 100% sure someone will ask about it

Reply:
Ok can we put it at the bottom then? That way it's more of an "afterthought".. as it should be imo.